paddy/writer.py

```diff
@@ -324,13 +333,15 @@ def clean_parameter_print(dirty_values):
     The function is typically called by `PFARunner.paddy_plot_and_print`
     when passed the argument 'final_results'.
     """
+    c = 0
     for i in dirty_values:
         temp = []
-        for j in dirty_values[i]['parameters']:
+        for j in dirty_values[str(c)]['parameters']:
```

Comment:
Anytime that you are using the `str` cast of an integer, you **need** to think "am I using the correct data structure?"